### ISSUE TITLE
[SPIKE] How do we support multiple parallel lines in the same distribution?

### ISSUE THREAD

**Issue by jmargutt** [rodekruis/emergency-response#46]
### **Who requested this spike**
Jeroen. 

### **Functional or Technical Spike**
Functional and technical.

### **User Story**
As a relief officer
I want to run multiple distribution lines in parallel, during the same distribution
So that I can be faster and serve more people

### **Acceptance Criteria**
- [ ] List possible functional and/or technical solutions, with pros and cons
- [ ] Choose one
- [ ] Create new issue to implement it

### **Timebox**
2-4h

**Comment by PanisMarijke** [comment]
@gal-agmon  please review and/or defer if you have moved this to another board.

**Comment by gal-agmon** [comment]
@PanisMarijke these are really old we never worked with GitHub on the new ReliefBox
